ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Query.hs

```diff
@@ -80,9 +103,17 @@ instance Eq (SomeSecond BlockQuery blk) => Eq (SomeSecond Query blk) where
   SomeSecond GetSystemStart == SomeSecond GetSystemStart = True
   SomeSecond GetSystemStart == _                         = False
 
+  SomeSecond GetChainBlockNo == SomeSecond GetChainBlockNo = True
+  SomeSecond GetChainBlockNo == _                        = False
+
+  SomeSecond GetChainPoint == SomeSecond GetChainPoint = True
+  SomeSecond GetChainPoint == _                      = False
+
 instance Show (SomeSecond BlockQuery blk) => Show (SomeSecond Query blk) where
   show (SomeSecond (BlockQuery blockQueryA)) = "Query " ++ show (SomeSecond blockQueryA)
   show (SomeSecond GetSystemStart) = "Query GetSystemStart"
+  show (SomeSecond GetChainBlockNo) = "Query GetChainBlockNo"
+  show (SomeSecond GetChainPoint) = "Query GetChainPoint"
```

Comment:
@Jasagredo, does your previous remark implies that we would want to align the definitions here as well?

Reply:
Aligned!